### PR TITLE
[Staging] Fix 404 on Document download

### PR DESCRIPTION
**Pull request by phillipahereza**
## Description

Catch `NotImplementedError` in FlourishView

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by phillipahereza** [comment]
@kilemensi 👀

File: takwimu/views.py [hunk 1]
```diff
@@ -57,7 +63,7 @@
 
 from .renderers import CSSRenderer
 from rest_framework.renderers import StaticHTMLRenderer
-from wagtail.documents.models import Document
+from wagtail.documents.models import Document, document_served
```
Comment (karimkawambwa) [inline takwimu/views.py]: where is `document_served` used ?

File: takwimu/views.py [hunk 2]
```diff
@@ -21,7 +24,10 @@
 from rest_framework import status
 
 from wagtail.contrib.settings.context_processors import settings
+from wagtail.core import hooks
 from wagtail.images import get_image_model
+from wagtail.utils import sendfile_streaming_backend
+from wagtail.utils.sendfile import sendfile
```
Comment (karimkawambwa) [inline takwimu/views.py]: where are these imports used?

File: Dockerfile [hunk 3]
```diff
@@ -1,4 +1,4 @@
-FROM codeforafrica/hurumap:0.6.3
+FROM codeforafrica/hurumap:0.6.2
```
Comment (kilemensi) [inline Dockerfile]: Are we also reverting the change in HURUmap `master`?

**Pull request merged**